pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -812,31 +814,41 @@ def show_help_instancename():
 """)
 
 
-def get_instancename(context, instancename, options, ns_names=None):
+def get_instancename(context, instancename, options, default_all_ns=False):
     """
     Common function to construct a CIMInstanceName object from the
     INSTANCENAME argument and the --key and --namespace options specified
     in the command line.
 
-    The keybindings of the returned CIM instance path must be specified in
+    If the ns_names option is a list, the request is to return instance
+    names in any of the namespaces.  Otherwise this function processes a
+    single namespace, the namespace defined by ns_names (string), the
+    namespace defined by the namespace component of the instancename or
+    the default namespace.
+
+    The keybindings of the returned CIM instance path(s) must be specified in
     exactly one of these ways:
 
     * If the keybindings component in the instance name string is "?"
       (e.g. "CIM_Foo.?"), the instances of that class are listed and the user is
-      prompted to pick one.
+      prompted to pick instance name. If multiple namespaces are specified
+      the instances in all namespaces are listed for the user to pick one.
 
     * If the "key" option is non-empty, the so specified key values are used as
       keybindings for the returned instance path, and the instance name string
       is interpreted as a class WBEM URI.
 
-    * If the instance name string specifies keybindings, they are used.
+    * If the instance name string specifies keybindings, these akeybindings
+      used as the instance name to get.
 
     The namespace of the returned CIM instance path must be specified in
     exactly one of these ways:
 
     * If the instance name string specifies a namespace, it is used.
 
-    * If the "namespace" option is non-empty, it is used.
+    * If the "namespace" option is non-empty, it is used. If the ns_names option
```

Comment:
1. There is no "ns_names" parameter anymore.
2. There is no "namespace" option.

Reply:
DONE